docs/DeveloperGuide.md

```diff
@@ -810,43 +800,134 @@ testers are expected to do more *exploratory* testing.
    1. Re-launch the app by double-clicking the jar file.<br>
        Expected: The most recent window size and location is retained.
 
-1. _{ more test cases …​ }_
+### 7.2 Adding a client
+
+1. Adding a client in the home window.
+
+   1. Test case: `add n/Steve Rogers p/91820392 e/thefirstavenger@test.com a/33 Apple Road, 928103`
+      Expected: New client with the details are added to the end of the list. Details of the added client are shown in
+                the status message and in the result display.
+   
+   2. Test case: `e/thefirstavenger@test.com add n/Steve Rogers a/33 Apple Road, 928103 p/91820392`
+      Expected: Similar to previous as the input order of the details does not matter.
+   
+   3. Test case: `add n/@$%^&&* p/91820392 e/thefirstavenger@test.com a/33 Apple Road, 928103`
+      Expected: No client is added. Error details shown in the status message and in the result display.
```

Comment:
<img width="774" alt="image" src="https://user-images.githubusercontent.com/77235338/140692945-26dbb126-2714-44a4-9783-75321819f994.png">
i think its more appropriate to say client is not added. what's the diff between status message and result display.